packages/store-ui/src/molecules/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,197 @@
+import type { PropsWithChildren } from 'react'
+import React, { useMemo } from 'react'
+import type { SwipeableProps } from 'react-swipeable'
+
+import Button from '../../atoms/Button'
+import Icon from '../../atoms/Icon'
+import { RightArrowIcon, LeftArrowIcon } from './Arrows'
+import useSlider from '../../hooks/useSlider/useSlider'
+import useSlideVisibility from './hooks/useSlideVisibility'
+import Bullets from '../Bullets'
+
+export interface CarouselProps extends SwipeableProps {
```

Comment:
What do you think about putting stick together the Component function definition?

Reply:
Sure!